### ISSUE TITLE
dtslint "Could not find implementations for the following rules"

### ISSUE THREAD

**Issue by maxlever** [DefinitelyTyped/DefinitelyTyped#61814]
Hello! This error shows up whenever I run `npm run test-all` or `npm run test my-package` (working on braintree-web in this case). Can I ignore this or does this mean my test didn't run?

```
dtslint@0.0.124

Could not find implementations for the following rules specified in the configuration:
    dt-header
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```

<details>

<summary>Expand for full test runs</summary>

```
➜  DefinitelyTyped git:(master) npm run test braintree-web

> definitely-typed@0.0.3 test /Users/maxlever/Code/DefinitelyTyped
> node --require source-map-support/register node_modules/@definitelytyped/dtslint/ types "braintree-web"

dtslint@0.0.124
npm run test-all
Could not find implementations for the following rules specified in the configuration:
    dt-header
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
        

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.8.0

YOUR TYPESCRIPT VERSION: 4.9.0-dev.20220818

Please only submit bug reports when using the officially supported version.

=============
➜  DefinitelyTyped git:(master) npm run test-all

> definitely-typed@0.0.3 test-all /Users/maxlever/Code/DefinitelyTyped
> node --require source-map-support/register node_modules/@definitelytyped/dtslint-runner/ --path .

dtslint-runner@0.0.124
Using local DefinitelyTyped at .
Parsing definitions...
Found 8210 packages.
Parsing in 10 child process(es)...
Parsing took 16.15 s
Running: git rev-parse --verify master
3d27049d07231e463d63c4728d174eab854e54a0
Running: git diff master --name-status

Running: git diff master~1 --name-status
M       types/use-position/index.d.ts
M       types/use-position/use-position-tests.ts
Testing 1 changed packages: use-position
Testing 0 dependent packages: 
Installing NPM dependencies...
1> use-position START
dtslint@0.0.124

Could not find implementations for the following rules specified in the configuration:
    dt-header
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
        
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.8.0

YOUR TYPESCRIPT VERSION: 4.9.0-dev.20220818

Please only submit bug reports when using the officially supported version.

=============
1> use-position OK


=== SUGGESTIONS ===

{}


=== PERFORMANCE ===

{ "braintree-web": 11515, "use-position": 1911}
  * Percentiles: 
99: 11515
95: 11515
90: 11515
70: 10554.599999999999
50: 6713
```

</details>

**Comment by sandersn** [comment]
Try merging from master; dt-header just got ported from tslint to eslint.

**Comment by maxlever** [comment]
It seems like this happens in the latest master too, according to the latest build: https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=137689&view=logs&j=8635df7d-ecde-52a7-c7a8-84b998e35690&t=c264d665-0b79-59b7-7e04-9965cbe74860&l=92.

However, it appears to be a warning since the build succeeded. Does that mean it should be ignored? If it doesn't affect the result of the test, we can close this ticket, but ideally we should get rid of misleading messages like this!

**Comment by sandersn** [comment]
It was an oversight in microsoft/DefinitelyTyped-tools#514, fixed in microsoft/DefinitelyTyped-tools#524, which shipped yesterday.
